core/src/main/java/org/elasticsearch/common/io/stream/StreamInput.java

```diff
@@ -762,4 +785,5 @@ public static StreamInput wrap(byte[] bytes) {
     public static StreamInput wrap(byte[] bytes, int offset, int length) {
         return new InputStreamStreamInput(new ByteArrayInputStream(bytes, offset, length));
     }
+
```

Comment:
It's from a suggestion I made in an earlier pull request. The codebase is not consistent but there is definitely a tendency towards having a blank line between the opening of a class and its body, and its body and the closing of a class. It's not a requirement (but I wish that it was).